modules/pytket-ionq/tests/backend_test.py

```diff
@@ -85,8 +85,9 @@ def test_invalid_request() -> None:
         assert "does not satisfy GateSetPredicate" in str(excinfo.value)
 
 
+@pytest.mark.skipif(skip_remote_tests, reason=REASON)
 def test_machine_debug() -> None:
-    b = IonQBackend(api_key="invalid", device_name="simulator", label="test 5")
+    b = IonQBackend(device_name="simulator", label="test 5")
```

Comment:
Why removing the `api_key`?

Reply:
It was a mistake

Reply:
With my changes an invalid API key provided to the backend will not allow the backend to instantiate so this test would not run

Reply:
OK but here you don't test for qubit info here. So your changes shouldn't modify the overall behaviour and especially not introducing a runtime error while the functionality is not tested for.